server/server.go

```diff
@@ -377,10 +375,13 @@ func New(
 		}
 	}
 
-	if s.appArmorEnabled && s.appArmorProfile == apparmorDefaultProfile {
-		if err := apparmor.InstallDefault(apparmorDefaultProfile); err != nil {
-			return nil, fmt.Errorf("ensuring the default apparmor profile %q is installed failed: %v", apparmorDefaultProfile, err)
+	if s.appArmorEnabled && config.ApparmorProfile == libconfig.DefaultApparmorProfile {
+		logrus.Infof("installing default apparmor profile: %v", libconfig.DefaultApparmorProfile)
+		if err := apparmor.InstallDefault(libconfig.DefaultApparmorProfile); err != nil {
+			return nil, fmt.Errorf("ensuring the default apparmor profile %q is installed failed: %v", libconfig.DefaultApparmorProfile, err)
 		}
+	} else {
+		logrus.Infof("assuming user-provided apparmor profile: %v", config.ApparmorProfile)
```

Comment:
I'm personally not sure, so thought I'd solicit opinions.  Should this logrus here and the one at line 379 be Infof or Debugf?

Reply:
I decided to go for the same verbosity of the seccomp profile log above and it only logs on server initialization. :) @vrothberg what do you think?

Reply:
I don't feel strongly on that. `Info` might add to the logs but it's a security-related one and only printed at start-up, so it seems reasonable to me.

Reply:
Since it follows the seccomp log then, let's leave it as Infof.  Thanks for entertaining the question.